[gatsby-transformer-documentationjs v2] Prismjs is a dependency but not shown in package.json

## Description

`gatsby-transformer-documentationjs` [requires prismjs](https://github.com/gatsbyjs/gatsby/blob/v2/packages/gatsby-transformer-documentationjs/src/gatsby-node.js#L10) but prismjs is not a dependency in its [package.json](https://github.com/gatsbyjs/gatsby/blob/v2/packages/gatsby-transformer-documentationjs/package.json)

### Steps to reproduce

```
git checkout v2
cd www
npm run develop
```

### Expected result

it should run without issue

### Actual result

```
error UNHANDLED EXCEPTION


  Error: Cannot find module 'prismjs'

  - v8-compile-cache.js:159 require
    [www]/[v8-compile-cache]/v8-compile-cache.js:159:20

  - gatsby-node.js:17 Object.<anonymous>
    [www]/[gatsby-transformer-documentationjs]/gatsby-node.js:17:15

  - v8-compile-cache.js:178 Module._compile
    [www]/[v8-compile-cache]/v8-compile-cache.js:178:30

  - v8-compile-cache.js:159 require
    [www]/[v8-compile-cache]/v8-compile-cache.js:159:20

  - api-runner-node.js:81 runAPI
    [www]/[gatsby]/dist/utils/api-runner-node.js:81:22

  - api-runner-node.js:173 mapSeries
    [www]/[gatsby]/dist/utils/api-runner-node.js:173:25

  - map.js:27
    [www]/[gatsby]/[async]/internal/map.js:27:9

  - eachOfLimit.js:66 replenish
    [www]/[gatsby]/[async]/internal/eachOfLimit.js:66:17


npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

### Environment

```

  System:
    OS: macOS Sierra 10.12.6
    CPU: x64 Intel(R) Core(TM) i7-4650U CPU @ 1.70GHz
    Shell: 5.2 - /bin/zsh
  Binaries:
    Node: 8.5.0 - ~/.nvm/versions/node/v8.5.0/bin/node
    Yarn: 1.6.0 - ~/.nvm/versions/node/v8.5.0/bin/yarn
    npm: 6.0.0 - ~/.nvm/versions/node/v8.5.0/bin/npm
  Browsers:
    Chrome: 67.0.3396.87
    Firefox: 58.0.1
    Safari: 11.0.1
  npmPackages:
    gatsby: next => 2.0.0-alpha.55 
    gatsby-image: next => 2.0.0-alpha.7 
    gatsby-plugin-canonical-urls: next => 1.0.14-12 
    gatsby-plugin-catch-links: next => 2.0.1-13 
    gatsby-plugin-feed: next => 1.3.19-13 
    gatsby-plugin-fullstory: ^1.0.4-5 => 1.0.4-9 
    gatsby-plugin-glamor: next => 2.0.0-alpha.15 
    gatsby-plugin-google-analytics: next => 1.0.20-12 
    gatsby-plugin-lodash: next => 3.0.0-alpha.14 
    gatsby-plugin-mailchimp: ^2.2.3 => 2.2.3 
    gatsby-plugin-manifest: next => 2.0.1-19 
    gatsby-plugin-netlify: next => 1.0.19-14 
    gatsby-plugin-nprogress: next => 2.0.0-alpha.14 
    gatsby-plugin-offline: next => 2.0.0-alpha.17 
    gatsby-plugin-react-helmet: next => 2.0.12-9 
    gatsby-plugin-sharp: next => 2.0.0-alpha.21 
    gatsby-plugin-sitemap: next => 1.2.14-12 
    gatsby-plugin-twitter: next => 1.0.18-13 
    gatsby-plugin-typography: next => 2.0.1-13 
    gatsby-remark-autolink-headers: next => 1.4.13-13 
    gatsby-remark-copy-linked-files: next => 1.5.28-12 
    gatsby-remark-images: next => 2.0.0-alpha.23 
    gatsby-remark-prismjs: next => 2.0.3-3 
    gatsby-remark-responsive-iframe: next => 1.4.18-12 
    gatsby-remark-smartypants: next => 1.4.12-13 
    gatsby-source-filesystem: next => 2.0.0-alpha.18 
    gatsby-source-npm-package-search: ^1.0.8 => 1.0.8 
    gatsby-transformer-csv: next => 1.3.9-12 
    gatsby-transformer-documentationjs: next => 1.4.10-12 
    gatsby-transformer-remark: next => 2.0.1-16 
    gatsby-transformer-sharp: next => 2.0.1-16 
    gatsby-transformer-yaml: next => 2.0.1-13 
  npmGlobalPackages:
    gatsby-cli: 1.1.58
```

--- a/packages/gatsby-transformer-documentationjs/package.json
+++ b/packages/gatsby-transformer-documentationjs/package.json
@@ -8,7 +8,8 @@
   },
   "dependencies": {
     "@babel/runtime": "7.0.0-beta.47",
-    "documentation": "^7.1.0"
+    "documentation": "^7.1.0",
+    "prismjs": "^1.14.0"
   },
   "devDependencies": {
     "@babel/cli": "7.0.0-beta.47",

